src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs

```diff
@@ -179,13 +179,13 @@ private unsafe QuicConnection()
         try
         {
             QUIC_HANDLE* handle;
-            ThrowHelper.ThrowIfMsQuicError(MsQuicApi.Api.ApiTable->ConnectionOpen(
-                MsQuicApi.Api.Registration.QuicHandle,
+            ThrowHelper.ThrowIfMsQuicError(MsQuicApi.Api.ConnectionOpen(
+                MsQuicApi.Api.Registration,
                 &NativeCallback,
                 (void*)GCHandle.ToIntPtr(context),
                 &handle),
```

Comment:
For the *Open methods we could make the wrapper use `out MsQuicContextSafeHandle handle` parameter and construct the safe handle inside. What do you think?

Reply:
I'll try it out and see if makes sense. If yes, I'll work it in. I like the idea.

Reply:
This would be hard to make work automagically (tm) with source generators. But we could introduce manual helpers. Once again, postponing this for a follow up.